### PR TITLE
#840 addmore faqs

### LINKED ISSUE #840: Add more and updated FAQs to the application.
More FAQs can be added to the application to provide more information regarding frequent concerns like:
*  What is the format of the images extracted from the PDF?
*  Can I compress the images before creating a PDF? 
    and many more.

### PR DESCRIPTION
**Pull request by rutujaac**
# Description

 To add more and updated FAQs.

Fixes #840 

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -565,6 +576,17 @@
         <item>No, you can only set a password for one file at a time.</item>
         <item>The software is only available in English language for now. Translations to other languages would be available soon. If you feel you can contribute to this software in terms of translation feel free to send a pull request.</item>
         <item>No, the developer does not collect any information when the software is utilized because the developer does not have access to any such information.</item>
+        <item>Yes, you can compress the images before using them to create a PDF by clicking on Image compression when using the Images to PDF feature.</item>
+        <item>The images extracted from a PDF are in png format.</item>
+        <item>No, files from cloud storage cannot be selected.</item>
```
Comment (Swati4star) [inline app/src/main/res/values/strings.xml]: but we are planning to add it soon!

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -565,6 +576,17 @@
...
+        <item>Yes, you can see all the files you have created/modified in the software. \nTo do this, click on History on the home page.</item>
+        <item>Yes, you can merge multiple PDF files together. Click on Merge PDFs available in the Modify existing PDFs on the home page.</item>
+        <item>Yes, you can edit and apply filters to the images before creating a PDF. \nTo do this use Edit Images and Filter Images options available while converting Images to PDF.</item>
+        <item>Yes, to add a watermark scroll down to Enhance Created PDFs on the home page and click on Add Watermark.</item>
+        <item>Yes, to change the sequence of pages in the PDF, select Reorder Pages from the home page.\nYou can use the arrow keys to order the pages or even delete one or more pages.</item>
+        <item>Yes, to extract images or text from a PDF, use Extract Images and Extract Text available on the home page.</item>
```
Comment (Swati4star) [inline app/src/main/res/values/strings.xml]: home page or from the navigation drawer

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -565,6 +576,17 @@
...
+        <item>Yes, you can use the enhancement options available in the Text to PDF option.</item>
+        <item>No, the PDF can only be converted to text format.</item>
```
Comment (Swati4star) [inline app/src/main/res/values/strings.xml]: but we are planning to add it soon!

**Comment by Swati4star** [comment]
can you also delete https://github.com/Swati4star/Images-to-PDF/blob/master/FAQs.md as it isnt relevant anymore.

**Comment by rutujaac** [comment]
Can you please review the changes?

**Pull request merged**